lightning/src/ln/channel.rs

```diff
@@ -803,9 +807,23 @@ macro_rules! secp_check {
 }
 
 impl<Signer: Sign> Channel<Signer> {
-	// Convert constants + channel value to limits:
-	fn get_holder_max_htlc_value_in_flight_msat(channel_value_satoshis: u64) -> u64 {
-		channel_value_satoshis * 1000 / 10 //TODO
+	/// Returns the value to use for `holder_max_htlc_value_in_flight_msat` as a percentage of the
+	/// `channel_value_satoshis` in msat, based on how many of the percent its been configured to
+	/// be set to by the user.
+	///
+	/// As we only allow the user to set the percentage to a value between `1-100`, if the user
+	/// sets a value less than `1`, the function will result in `1` percent of the
+	/// `channel_value_satoshis`, as well as `100` percent if the user has set a value larger
+	/// than `100`.
```

Comment:
Can replace this paragraph with: "The effective percentage is lower bounded by 0% and upper bounded by 100%."

Reply:
Thanks! I figured you probably ment "lower bounded by 1%", so I edited to that, but let me know if I misunderstood :).